components/playground/main.go

```diff
@@ -123,9 +121,27 @@ Examples:
 		SilenceUsage: true,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			if len(args) > 0 {
+				// Any way to add subcommand but still compatible with the old usage?
```

Comment:
We can add `scale-in/scale-out/display` subcommands to the rootCmd to avoid manually handle the first argument.

Reply:
then we can't not run `tiup playground v3.0.0` and more.
some error like "no command v3.0.0..", it will treat v3.0.0 as a subcommand too.
I am not sure if there' anyway to make it still run the root cmd if no subcommand match.

Reply:
A workaround is to add a subcommand like `start` work as the origin rootcmd, and add `start` in args[1] if args[1] not match any subcommand before calling `Execute` of rootcmd

https://github.com/spf13/cobra/issues/725#issuecomment-411807394